vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/dnd/GridDragEndEvent.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2000-2019 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.grid.dnd;
+
+import com.vaadin.flow.component.ComponentEvent;
+import com.vaadin.flow.component.DomEvent;
+import com.vaadin.flow.component.grid.Grid;
+
+/**
+ * Drop event on an HTML5 drop target {@link Grid} row.
```

Comment:
Weird sentence

Reply:
From https://github.com/vaadin/framework/blob/master/server/src/main/java/com/vaadin/ui/components/grid/GridDragEndEvent.java#L26

Reply:
Do you think the javadoc is good? IMO it's not, and there's no need to carry bad javadocs from the old framework.

At least it should be fixed to say "Drag end event" instead of "Drop event"

Reply:
Fixed